### PR TITLE
remove subnet locals and put the name values into the config 

### PR DESCRIPTION
**Pull request by betsy-lichtenberg**

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

File: examples/basic_custom_mode/main.tf [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+provider "google" {
+  version = "~> 3.45.0"
+}
+
+# [START vpc_custom_create]
+resource "google_compute_network" "vpc_network" {
+  project                 = var.project_id
+  name                    = var.network_name
```
Comment (morgante) [inline examples/basic_custom_mode/main.tf:29]: I recommend simply hard-coding this name.

File: examples/basic_auto_mode/main.tf [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+provider "google" {
+  version = "~> 3.45.0"
+}
+
+# [START vpc_auto_create]
+resource "google_compute_network" "vpc_network" {
+  project                 = var.project_id
+  name                    = var.network_name
```
Comment (morgante) [inline examples/basic_auto_mode/main.tf:29]: We can hard-code this name.

File: examples/basic_custom_mode/main.tf (hunk 1 above)
Comment (betsy-lichtenberg) [reply examples/basic_custom_mode/main.tf:29]: Done

File: examples/basic_auto_mode/main.tf (hunk 2 above)
Comment (betsy-lichtenberg) [reply examples/basic_auto_mode/main.tf:29]: Done

**Pull request closed without merging**